Networking/HTTP/HTTPLogic.cc

```diff
@@ -93,7 +93,7 @@ namespace litecore { namespace net {
             if (_httpStatus == HTTPStatus::ProxyAuthRequired)
                 Assert(_proxy && _proxy->username);
             else
-                Assert(_authHeader);
+                Assert(_enableChallengeAuth && _authHeader);
```

Comment:
Sorry it's this assertion...

Reply:
It will be hit only when there is a challenge. So if _enableChallengeAuth == false, it should be hit at all.